src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesSpec.scala

```diff
@@ -86,16 +80,89 @@ class ResourceRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest
   }
 
   it should "404 on list resource roles when resource type doesnt exist" in {
-    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceAction("run")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
+    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceActionPattern("run")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
     val samRoutes = TestSamRoutes(Map(resourceType.name -> resourceType))
 
     Get(s"/api/resource/doesntexist/foo/roles") ~> samRoutes.route ~> check {
       status shouldEqual StatusCodes.NotFound
     }
   }
 
-  "PUT /api/resource/{resourceType}/{resourceId}/policies" should "201 on a new policy being created for a resource" in {
-    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceAction("alter_policies"), ResourceAction("can_compute")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("alter_policies")))), ResourceRoleName("owner"))
+  "GET /api/resource/{resourceType}/{resourceId}/policies/{policyName}" should "200 on existing policy of a resource with read_policies" in {
+    val members = AccessPolicyMembership(Set(defaultUserInfo.userEmail), Set(ResourceAction("can_compute")), Set.empty)
```

Comment:
As far as I can tell, all references to "can_compute" and "canCompute" are not significant to these tests. For clarity, I would rather they not be there. To verify my thinking, I went ahead and made those changes and created a PR if that's helpful: https://github.com/broadinstitute/sam/pull/74.